Last-Update: 2015-11-12
Forwarded: http://www.midnight-commander.org/ticket/3553
Bug-MC: http://www.midnight-commander.org/ticket/3553
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fix FTBFS in 'make update-po'.

--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -10,11 +10,9 @@
 lib/tty/key.c
 lib/tty/tty.c
 lib/tty/tty-slang.c
 lib/utilunix.c
-lib/vfs/dir/entry.c
 lib/vfs/direntry.c
-lib/vfs/dir/utils.c
 lib/vfs/interface.c
 lib/vfs/parse_ls_vga.c
 lib/vfs/utilvfs.c
 lib/widget/dialog-switch.c
